src/index.ts

```diff
@@ -68,7 +68,12 @@ export interface IClientConfig {
   pollAfterFailedInitialization?: boolean;
 }
 
-export { IAssignmentEvent, IAssignmentLogger } from '@eppo/js-client-sdk-common';
+export {
+  IAssignmentEvent,
+  IAssignmentLogger,
+  IBanditEvent,
+  IBanditLogger,
+} from '@eppo/js-client-sdk-common';
```

Comment:
I wonder if we also want to expose any of the following:
* `Attributes` - used for subject attributes
* `ContextAttributes` - used with bandits when explicitly separating out numerical and categorical attributes
* `BanditSubjectAttributes - used with bandits
* `BanditActions` - used with bandits

Link to types: https://github.com/Eppo-exp/js-client-sdk-common/blob/main/src/types.ts

Example method signatures:

```ts
getBanditAction(
  flagKey: string,
  subjectKey: string,
  subjectAttributes: BanditSubjectAttributes,
  actions: BanditActions,
  defaultValue: string,
): { variation: string; action: string | null };
```

Reply:
sounds like yes, if those are part of the public API!

Reply:
These are my thoughts as well. 

Note this requires a change to the common SDK. Pr for that here: https://github.com/Eppo-exp/js-client-sdk-common/pull/114